### PR TITLE
Navbar newsfeed link

### PR DESCRIPTION
**Pull request by jransome**
"acebook" in the navbar now links to the news_feed/index (though style needs updating). Author names now also appear next to posts in the news feed.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <%= javascript_include_tag 'application' %>
   </head>
   <nav class="navbar navbar-light bg-light justify-content-between">
-    <a class="navbar-brand">A C E B O O K</a>
+    <a class="navbar-brand"><%= link_to("A C E B O O K", "news_feed/index") %></a>
```
Comment (dearshrewdwit) [inline app/views/layouts/application.html.erb]: Use a path helper! `newsfeed_path` 💃  or something like that

File: app/views/news_feed/index.html.erb [hunk 2]
```diff
@@ -2,7 +2,8 @@
 
 <% @activities.each do |activity| %>
   <div class = "activity">
-  [username here]<%= link_to activity.owner.name, activity.owner if activity.owner %>
+  <%= "#{activity.trackable.user.first_name} #{activity.trackable.user.last_name}" %>
```
Comment (dearshrewdwit) [inline app/views/news_feed/index.html.erb:5]: Is there a way to not have to break the law of demeter here? can we use a model method somewhere to return the full name of a user?

File: spec/features/author_name_appears_with_posts_in_news_feed_spec.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+require 'user_helpers'
+
+RSpec.feature "Username with posts in news feed", type: :feature do
+  let(:first_name) { "Ed" }
```
Comment (dearshrewdwit) [inline spec/features/author_name_appears_with_posts_in_news_feed_spec.rb:5]: 👀

File: spec/features/author_name_appears_with_posts_in_news_feed_spec.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  let(:last_name) { "Withers" }
+
+  before do
+    sign_up(first_name: first_name, last_name: last_name)
+    visit "/posts"
```
Comment (dearshrewdwit) [inline spec/features/author_name_appears_with_posts_in_news_feed_spec.rb:10]: `posts_path`

File: spec/features/author_name_appears_with_posts_in_news_feed_spec.rb [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    within (".new_post") do
+      fill_in "post_message", with: "Hello, world!"
+      click_button "Submit"
+    end
+
+  end
+
+  scenario "when posts appear in the news feed" do
+    visit "/news_feed/index"
```
Comment (dearshrewdwit) [inline spec/features/author_name_appears_with_posts_in_news_feed_spec.rb:19]: `news_feed_path` or something

File: spec/features/news_feed_is_linked_to_navbar_spec.rb [hunk 6]
```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.feature "Newsfeed link in navbar", type: :feature do
+  before do
+    sign_up
+  end
+
+  scenario "on any page" do
+    click_link "A C E B O O K"
+    expect(current_path).to eq("/news_feed/index")
```
Comment (dearshrewdwit) [inline spec/features/news_feed_is_linked_to_navbar_spec.rb:10]: .... rails

File: spec/features/newsfeed_is_linked_to_navbar_spec.rb [hunk 7]
```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.feature "Newsfeed link in navbar", type: :feature do
+  before do
+    sign_up
+  end
+
+  scenario "on any page" do
+    click_link "A C E B O O K"
+    expect(current_path).to eq("/news_feed/index")
```
Comment (dearshrewdwit) [inline spec/features/newsfeed_is_linked_to_navbar_spec.rb:10]: ..........

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]

**Comment by dearshrewdwit** [comment]
Why is the route: `news_feed_index GET    /news_feed/index(.:format)   `

**Comment by dearshrewdwit** [comment]
```
Rails.application.routes.draw do

  get 'news_feed/index'
  devise_for :users

  resources :users do
    resources :albums
  end

  resources :posts do
    resources :comments
    resources :likes
  end

  root to: 'posts#index'
end

```

What should it actually be?

**Comment by aliwoodman** [comment]
These changes have been made but in a new branch elegantly named "combined-navbar-link-and-newsfeed-shows-likes" (that branch was a combination of two other branches...ahem)

**Pull request closed without merging**